frontend/src/features/permits/apiManager/permitsAPI.ts

```diff
@@ -7,22 +7,36 @@ import {
   httpGETRequestStream,
 } from "../../../common/apiManager/httpRequestHandler";
 
-import { getDefaultRequiredVal, replaceEmptyValuesWithNull } from "../../../common/helpers/util";
-import { Application, ApplicationResponse, PermitApplicationInProgress } from "../types/application";
+import {
+  getDefaultRequiredVal,
+  replaceEmptyValuesWithNull,
+} from "../../../common/helpers/util";
+import {
+  Application,
+  ApplicationResponse,
+  PermitApplicationInProgress,
+} from "../types/application";
 import { DATE_FORMATS, toLocal } from "../../../common/helpers/formatDate";
 import { APPLICATION_UPDATE_STATUS_API, PAYMENT_API, PERMITS_API } from "./endpoints/endpoints";
 import { mapApplicationToApplicationRequestData } from "../helpers/mappers";
 import { PermitTransaction, Transaction } from "../types/payment";
 import { VEHICLES_URL } from "../../../common/apiManager/endpoints/endpoints";
+import { ReadPermitDto } from "../types/permit";
+
+/**
+ * A record containing permit keys and full forms.
+ */
+const permitAbbreviations: Record<string, string> = {
```

Comment:
I think there's a method under permits/helpers/mappers.ts called "permitTypeDisplayText" that does something similar to this

Reply:
Discussed off github but the gist is that there is difference in terminology between `permitTypeDisplayText` function and the object here. Another issue was that it needs a type change inside `PermitApplicationInProgress` for the `permitType` field. This can have other unwanted ramifications and between reverting to the code in main and the one in this PR, we decided to stick with this PR's version.